userspace/libscap/scap.c

```diff
@@ -269,7 +269,9 @@ scap_t* scap_open_live_int(char *error,
 	if((res = scap_proc_scan_proc_dir(handle, filename, -1, -1, NULL, error, true)) != SCAP_SUCCESS)
 	{
 		scap_close(handle);
-		snprintf(error, SCAP_LASTERR_SIZE, "error creating the process list. Make sure you have root credentials.");
+		char err[SCAP_LASTERR_SIZE] = {0};
```

Comment:
from [snprintf(3)](https://linux.die.net/man/3/snprintf)

> The functions snprintf() and vsnprintf() do not write more than size bytes (including the terminating null byte ('\0')).

If we want to be extra careful, there are many places where this should be corrected.